### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

- Closes #258
- Closes #259
- Closes #260
- Closes #261
- Closes #262 
- Closes #263

We have other outdated deps mostly post-css related. Do we know why we don't update them?

### CODE REVIEW

File: packages/application-shell/src/components/login/countdown.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ export class Countdown extends React.PureComponent {
 
   static propTypes = {
     onCountEnd: PropTypes.func.isRequired,
+    children: PropTypes.func.isRequired,
```
[the remark this answers is not in the document]
Comment (emmenko) [reply packages/application-shell/src/components/login/countdown.js]: Interesting...was that a bug in eslint?

[the remark this answers is not in the document]
Comment (tdeekens) [reply packages/application-shell/src/components/login/countdown.js]: I assume so yes.

**Comment by tdeekens** [comment]
@emmenko any objection or can you approve?

**Comment by emmenko** [comment]
> We have other outdated deps mostly post-css related. Do we know why we don't update them?

Not that I know of. Which deps are you talking about?

**Comment by tdeekens** [comment]
cleanshot 2019-01-08 at 10 41 30 2x

I remember that we can update some of these but not sure about all.

**Comment by emmenko** [comment]
I see...hmm I guess this would require a bit more work/testing.

**Comment by tdeekens** [comment]
I updated the webpack loaders too. According to their changelogs I can't spot breaking changes affecting us.

**Comment by tdeekens** [comment]
...and `unfetch` too now.

**Pull request merged**